advanced/mathfx/src/main/java/org/arakhne/afc/math/geometry/d2/ifx/MultiShape2ifx.java

```diff
@@ -231,6 +231,23 @@ public void invalidated(Observable observable) {
 			}
 		}
 
+		public boolean equals(Object obj) {
+			if (!super.equals(obj)) {
+				return false;
+			}
+			final InternalObservableList lObj = (InternalObservableList) obj;
+			if (internalList.equals(lObj.internalList)) {
+				return internalList.equals(lObj.internalList);
+			}
+			return false;
+		}
+
+		public int hasCode() {
```

Comment:
Sorry for the silly mistake

On Thu, Jun 2, 2016 at 1:47 PM, Stéphane Galland <notifications@github.com>
wrote:

> In
> advanced/mathfx/src/main/java/org/arakhne/afc/math/geometry/d2/ifx/MultiShape2ifx.java
> <https://github.com/gallandarakhneorg/afc/pull/65#discussion_r65518804>:
>
> > @@ -231,6 +231,23 @@ public void invalidated(Observable observable) {
> >  			}
> >  		}
> >
> > +		public boolean equals(Object obj) {
> > +			if (!super.equals(obj)) {
> > +				return false;
> > +			}
> > +			final InternalObservableList lObj = (InternalObservableList) obj;
> > +			if (internalList.equals(lObj.internalList)) {
> > +				return internalList.equals(lObj.internalList);
> > +			}
> > +			return false;
> > +		}
> > +
> > +		public int hasCode() {
>
> The name of the function is wrong. It must be hashCode.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/65/files/578c7c9af6dee2448eb28b43d8b96f4649944be9#r65518804>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BITZqEMx8vSmkQY_zbIbTd4QlXcAks5qHrTDgaJpZM4IptzN>
> .
>